modules/java/examples/code_snippets/Examples.java

```diff
@@ -3142,30 +3142,68 @@ public void testQuerySyntaxId() throws CouchbaseLiteException {
     public void testQueryPagination() throws CouchbaseLiteException {
 
 
-        // tag::query-syntax-pagination[]
-        try {
-          this_Db = new Database("hotels");
-        } catch (CouchbaseLiteException e) {
-          e.printStackTrace();
-        }
+      // tag::query-syntax-pagination[]
+      try {
```

Comment:
In Kotlin code, you can probably just leave the try/catch block out.  ... especially one that doesn't do any more than print the stack trace and ignore the error

Reply:
Will do